src/types/claim/resolvers.js

```diff
@@ -19,6 +19,12 @@ module.exports = {
   Claim: {
     references: (_) => {
       return _.references;
+    },
+    qualifier: (_, { propertyId }) => {
```

Comment:
```suggestion
    qualifiers: (_, { propertyId }) => {
```
This should be plural, I think.

We could also make the argument optional and return all qualifiers if propertyId is not provided. That could happen in a separate patch though.